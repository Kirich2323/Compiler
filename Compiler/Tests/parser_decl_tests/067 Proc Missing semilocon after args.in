procedure name(a : integer)
begin
end;