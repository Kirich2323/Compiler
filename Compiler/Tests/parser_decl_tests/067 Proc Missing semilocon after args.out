(2 ; 1): Got unexpected token: "begin". Expected: ";".